packages/core/src/types/Matrix.ts

```diff
@@ -1,4 +1,14 @@
-import {Vector2} from './Vector';
+import {Matrix2D} from './Matrix2D';
+import {PossibleVector2, Vector2} from './Vector';
+
+export * from './Matrix2D';
+
+export const EPSILON = 0.000001;
+
+export type PossibleMatrix2D =
+  | Matrix2D
+  | [number, number, number, number, number, number]
+  | [PossibleVector2, PossibleVector2, PossibleVector2];
```

Comment:
`undefined` should also be a possible matrix:
```suggestion
export type PossibleMatrix2D =
  | Matrix2D
  | [number, number, number, number, number, number]
  | [PossibleVector2, PossibleVector2, PossibleVector2]
  | undefined;
```